client/components/filters/compare/index.js

```diff
@@ -0,0 +1,116 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import { Button } from '@wordpress/components';
+import { Component } from '@wordpress/element';
+import PropTypes from 'prop-types';
+
+/**
+ * Internal dependencies
+ */
+import Card from 'components/card';
+import Search from 'components/search';
+import { updateQueryString } from 'lib/nav-utils';
+
+/**
+ * Displays a card + search used to filter results as a comparison between objects.
+ */
+class CompareFilter extends Component {
+	constructor( { getLabels, param, query } ) {
+		super( ...arguments );
+		this.state = {
+			selected: [],
+		};
+
+		this.updateQuery = this.updateQuery.bind( this );
+		this.updateLabels = this.updateLabels.bind( this );
+
+		if ( query[ param ] ) {
+			getLabels( query[ param ] ).then( this.updateLabels );
+		}
+	}
+
+	componentDidUpdate( { param } ) {
+		if ( param !== this.props.param ) {
```

Comment:
You'll need to compare `query[ param ]` also because the `&product=50%2C48%2C46` can be changed in two places:

![image](https://user-images.githubusercontent.com/1922453/45187131-c7f54500-b283-11e8-8a82-37c78619a0d3.png)

Reply:
yeah, the table compare button is my next task, my thinking was that I'd update this component as needed once I figure out how that'll work.